dev/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensiondev",
     "author": "Microsoft Corporation",
-    "version": "0.5.0",
+    "version": "0.5.1",
```

Comment:
Should this be a major update (i.e. `0.6.0`)?

Reply:
I'll go ahead and rev it then. It costs nothing and might save somebody from headaches in the future.